gen-intr-lut.sh

```diff
@@ -7,12 +7,10 @@ echo     "pub fn lookup_vector(chip: &str, intr: &str) -> Option<usize> {"
 echo     "    match chip {"
 
 for intr_path in "$@"; do
-    chip="$(basename "$(dirname "$intr_path")")"
+    chip="$(basename "${intr_path%%.*}")"
```

Comment:
This is a bit non-obvious.  I'd rather make it explicit what we're removing:

```suggestion
    chip="$(basename "${intr_path%.svd.patched}")"
```